packages/teams-js/src/private/files.ts

```diff
@@ -296,6 +296,105 @@ export namespace files {
     extension: string;
   }
 
+  /**
+   * @hidden
+   * Hide from docs
+   *
+   * Actions specific to 3P cloud storage provider file and / or account
+   */
+  export enum CloudStorageProviderFileAction {
+    New = 'NEW',
+    Rename = 'RENAME',
+    Download = 'DOWNLOAD',
+    Upload = 'UPLOAD',
+    Delete = 'DELETE',
+    Logout = 'LOGOUT',
+    Move = 'MOVE',
+    Copy = 'COPY',
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   *
+   * Interface for 3P cloud storage provider request content type
+   */
+  export interface I3PCloudStorageProviderRequest<T> {
+    content: T;
+  }
+
+  /**
+   * Base interface for 3P cloud storage provider action request content
+   */
+  interface IBase3PCloudStorageProviderContent {
```

Comment:
We generally don't prefix interfaces with "I" in the SDK and that's not really common in JavaScript. I would recommend omitting it. @erinha, @TrevorJoelHarris, thoughts? Do we have any formal guidance on this?

Reply:
Initially was referring to private **files namespace** and noticed the convention of prefixing interfaces with letter "I".
After your comment referred other namespaces and in most places the convention followed is to avoid prefix "I".
Removed the prefix "I" from interfaces i created.

Reply:
I'm fine with no I*  prefix.